### PR TITLE
[Vulkan] Add scripts for building vulkan packages

### PR DESCRIPTION
**Pull request by yzh119**

### CODE REVIEW

File: vulkan/docker/install/almalinux_install_conda.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+set -o pipefail
+
+cd /tmp && wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+chmod +x Miniconda3-latest-Linux-x86_64.sh
+/tmp/Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda
+rm /tmp/Miniconda3-latest-Linux-x86_64.sh
+/opt/conda/bin/conda upgrade --all
+/opt/conda/bin/conda clean -ya
+/opt/conda/bin/conda install conda-build conda-verify
+chmod -R a+w /opt/conda/
```
Comment (junrushao) [inline vulkan/docker/install/almalinux_install_conda.sh:14]: You might want to set the default solver to libmamba to accelerate conda solving:

```
conda update --yes -n base -c defaults conda
conda install -n base conda-libmamba-solver
conda config --set solver libmamba
```

File: vulkan/docker/Dockerfile.package-vulkan [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Install Conda
+COPY install/almalinux_install_conda.sh /install/almalinux_install_conda.sh
+RUN bash /install/almalinux_install_conda.sh
+ENV PATH=/opt/conda/bin:${PATH}
+
+# Install Python
+RUN conda create -n py37 python=3.7 -y
+RUN conda create -n py38 python=3.8 -y
+RUN conda create -n py39 python=3.9 -y
+RUN conda create -n py310 python=3.10 -y
```
Comment (junrushao) [inline vulkan/docker/Dockerfile.package-vulkan:30]: we might need py311 given it's been quite mature

File: vulkan/docker/install/almalinux_install_conda.sh (hunk 1 above)
Comment (yzh119) [reply vulkan/docker/install/almalinux_install_conda.sh:14]: thank you!

File: vulkan/docker/Dockerfile.package-vulkan (hunk 2 above)
Comment (yzh119) [reply vulkan/docker/Dockerfile.package-vulkan:30]: sounds good

**Comment by yzh119** [comment]
Note: [orderedset](https://github.com/simonpercivall/orderedset/issues/36) is not actively maintained and we removed it from the dependency to support python 3.11

**Review by junrushao — APPROVED** [review]

**Pull request merged**